pkg/csprofiles/csprofiles.go

```diff
@@ -31,8 +33,8 @@ func NewProfile(profilesCfg []*csconfig.ProfileCfg) ([]*Runtime, error) {
 	profilesRuntime := make([]*Runtime, 0)
 
 	for _, profile := range profilesCfg {
-		var runtimeFilter, runtimeDurationExpr *vm.Program
-		var debugFilter, debugDurationExpr *exprhelpers.ExprDebugger
+		var runtimeFilter, runtimeDurationExpr, notificationFilter *vm.Program
```

Comment:
Rename notificationFilter to runtimeNotificationFilter to fit naming conventions?